cli/src/main/resources/help/server.adoc

```diff
@@ -0,0 +1,67 @@
+SERVER(1)
+=========
+:doctype: manpage
+
+
+NAME
+----
+server - server configuration and state management.
+
+
+DESCRIPTION
+-----------
+
+
+
+SYNOPSIS
+--------
+*server report*
+
+*server connector ls*
+
+*server connector describe* 'connector-name'
+
+*server connector start* 'connector-name'
+
+*server connector stop* 'connector-name'
+
+*server connector ipfilter ls* 'connector-name'
+
+*server connector ipfilter set* 'connector-name' --rules='[ACCEPT|REJECT]/cidr',...
+
+*server connector ipfilter clear* 'connector-name'
+
+
+SERVER CONNECTOR IPFILTER OPTIONS
+---------------------------------
+
+*--rules*='[ACCEPT|REJECT]/cidr',...::
+One or more IP filtering rules.
+
+
+EXAMPLES
+--------
+
+`server report` +
+Obtains a server report, including information about network, threads, memory, etc.
+
+`server connector ls` +
+Lists all available connectors on the server.
+
+`server connector describe endpoint-default` +
+Shows information about the specified connector, including host, port, local and global connections, IP filtering rules.
+
+`server connector start endpoint-default` +
+Starts a connector so that it can accept connections across the cluster. This operation can only be performed on connectors which are directly bound to a socket.
+
+`server connector stop endpoint-default` +
+Stops a connector dropping all established connections across the cluster. This operation can only be performed on connectors which are directly bound to a socket.
+
+`server connector ipfilter ls endpoint-default` +
+Lists all IP filtering rules active on a connector across the cluster.
+
+`server connector ipfilter set endpoint-default --rules=ACCEPT/192.168.0.0/16,REJECT/10.0.0.0/8`
+Sets IP filtering rules on a connector across the cluster. Replaces all existing rules.
+
+`server connector ipfilter ls endpoint-default` +
```

Comment:
example already exist in line 60. Clear is missing.

Reply:
should be fixed now